#
# Definition of all trackable objects
# Identifier corresponds to Trackable ID set in Tracking Tools
#
rigid_bodies:
    '2': # = "User ID" of rigid body in Motive v1.0, or "Trackable ID" in Motive v2.0 
        pose: rob_2/pose # topic name of the published pose
        child_frame_id: base_link
        parent_frame_id: map
#    '3':
#        pose: rob_3/pose
#        child_frame_id: base_link
#        parent_frame_id: map
#    '4':
#        pose: rob_4/pose
#        child_frame_id: base_link
#        parent_frame_id: map
#    '5':
#        pose: rob_5/pose
#        child_frame_id: base_link
#        parent_frame_id: map

optitrack_config:
#        multicast_address: 224.0.0.1
        # Added by Eric Cristofalo 20170124
        multicast_address: 239.255.42.99
